### PR TITLE
AUT-1831 fix: action icons selection and behaviour

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-1831

**Steps to reproduce:**

- Go to Tests tab
- Create a new test and go to it's Authoring
- Add subsections up to 3rd level
- Click 'Move Down' (arrow down icon) on the 2nd level section

**Actual result:** All nested section are hidden.
**Expected result:** Nothing happened since there's no other sections and the icon is disabled.

### CODE REVIEW

**Review by Silvia-loza — APPROVED** [review]
- [X] New code is covered by tests (if applicable)
- [X] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X]  Pull request's target is not ` master`
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

File: views/js/controller/creator/views/actions.js [hunk 1]
```diff
@@ -107,14 +107,14 @@ define([
...
+            //prevent default and click during animation and on disabled icon
             e.preventDefault();
-            if ($element.is(':animated')) {
+            if ($element.is(':animated') && $element.hasClass('disabled')) {
                 return false;
             }
 
             //get the position
-            $elements = $('.' + elementClass, $container);
+            $elements =  $container.children(`.${elementClass}`);
```
Comment (yohanlaborda) [inline views/js/controller/creator/views/actions.js:117]: Additional blank space

File: views/js/test/creator/helpers/validators/test.js [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018-2022 (original work) Open Assessment Technologies SA ;
+ */
+/**
+ * @author Bertrand Chevrier <bertrand@taotesting.com>
+ */
+define([
+
```
Comment (yohanlaborda) [inline views/js/test/creator/helpers/validators/test.js:22]: Additional blank space

**Review by yohanlaborda — CHANGES_REQUESTED** [review]

**Review by vilka27 — APPROVED** [review]
LGTM

**Review by jsconan — APPROVED** [review]
It looks ok. Good job.
Only a few non blocking remarks.

**Review Checklist**
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Pull request merged**